src/Compilers/CSharp/Portable/Syntax/RecordDeclarationSyntax.cs

```diff
@@ -13,5 +13,13 @@ public partial class RecordDeclarationSyntax
                 return BaseList?.Types.FirstOrDefault() as PrimaryConstructorBaseTypeSyntax;
             }
         }
+
+        public RecordDeclarationSyntax Update(SyntaxList<AttributeListSyntax> attributeLists, SyntaxTokenList modifiers, SyntaxToken keyword, SyntaxToken identifier,
+            TypeParameterListSyntax? typeParameterList, ParameterListSyntax? parameterList, BaseListSyntax? baseList, SyntaxList<TypeParameterConstraintClauseSyntax> constraintClauses,
+            SyntaxToken openBraceToken, SyntaxList<MemberDeclarationSyntax> members, SyntaxToken closeBraceToken, SyntaxToken semicolonToken)
+        {
+            return Update(attributeLists, modifiers, keyword, recordModifier: default(SyntaxToken), identifier,
+                typeParameterList, parameterList, baseList, constraintClauses, openBraceToken, members, closeBraceToken, semicolonToken);
```

Comment:
this feels wrong.  if this is an Update call, then mi would expect to maintain the original class/struct mod, not blow it away.